src/test/java/ru/rosatom/documentflow/services/impl/DocumentProcessServiceImplTest.java

```diff
@@ -29,22 +32,74 @@ class DocumentProcessServiceImplTest {
     private final DocProcessRepository docProcessRepository = Mockito.mock(DocProcessRepository.class);
     private final EmailService emailService = Mockito.mock(EmailService.class);
     private final Producer producer = Mockito.mock(Producer.class);
-
     private final DocumentProcessService documentProcessService = new DocumentProcessServiceImpl(
             documentService,
             userService,
             docProcessRepository,
             emailService,
             producer);
 
+    @Nested
+    class ApprovalTests {
+        private DocProcess updatableDocProcess;
```

Comment:
Почему ApprovalTests если это тесты перевода в WAITING_FOR_APPROVE? Approval это законченное действие, т.е подтвержение, перевод в APPROVED

```suggestion
    class SendToApproveTests {
        private DocProcess updatableDocProcess;
```